Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp

```diff
@@ -1617,6 +1637,16 @@ void NetworkProcessProxy::increaseQuota(PAL::SessionID sessionID, const WebCore:
     });
 }
 
+void NetworkProcessProxy::sendStorageAccessPromptQuirksData()
+{
+#if ENABLE(ADVANCED_PRIVACY_PROTECTIONS)
+    if (StorageAccessPromptQuirkController::shared().cachedQuirks().isEmpty())
+        return;
+
+    send(Messages::NetworkProcess::SetStorageAccessPromptQuirks(StorageAccessPromptQuirkController::shared().cachedQuirks()), 0);
```

Comment:
We should probably just call this `UpdateStorageAccessPromptQuirks`, if it is only used for updating.